src/io_intf.ml

```diff
@@ -27,6 +27,17 @@ module type S = sig
   val rename : src:t -> dst:t -> unit
 
   val append : t -> string -> unit
+  (** [append io s] will automatically call [flush io] when a certain limit of
+      offset has been reached. This can flush some incomplete data if a crash
+      occurs during it. If you want to gain manual control over it, please use
+      [append_batch]. *)
```

Comment:
I think it makes sense to remove it altogether, and use append_batch everywhere.